bears/ruby/RuboCopBear.py

```diff
@@ -19,8 +19,7 @@ class RuboCopBear:
     """
 
     LANGUAGES = {'Ruby'}
-    REQUIREMENTS = {GemRequirement('rubocop'),
-                    PipRequirement('pyyaml', '3.12')}
```

Comment:
This was done because the shell foo in .travis.yml needs to find non-Python bears.  It is using the presence of `PipRequirement`, and there are two non-Python bears which use PipRequirement.

.travis.yml logic needs to be rewritten, and then this isnt necessary.